Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -104,7 +104,7 @@ open class _ChatChannelListVC<ExtraData: ExtraDataTypes>: _ViewController,
         loadingIndicator.pin(anchors: [.centerX, .centerY], to: view)
     }
     
-    override public func defaultAppearance() {
+    override open func setUpAppearance() {
```

Comment:
We need to call `super` for `setUpAppearance` now

Reply:
✅